org.eclipse.xtext/src/org/eclipse/xtext/resource/containers/StateBasedContainer.java

```diff
@@ -12,6 +12,7 @@
 import java.util.Collection;
 import java.util.Collections;
 import java.util.Map;
+import java.util.function.Function;
```

Comment:
@szarnekow do we have any consistent concept when to use
`java.util.function.Function` or  `org.eclipse.xtext.xbase.lib.Functions.Function1`
or `com.google.common.base.Function`

Reply:
Personally I have a preference towards `j.u.f.Function` whenever possible.